spec/features/accessibility/idv_pages_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'rails_helper'
 require 'axe/rspec'
 
-feature 'Accessibility on IDV pages', :js do
+feature 'Accessibility on IDV pages', :js, :accessibility do
```

Comment:
My first reaction is that I don't typically tag tests, so speaking on behalf of myself, I would forget to tag tests. There's also the `--pattern` which includes files so if we *only* wanted to run accessibility tests we could do that too